File: 프로그래머스/unrated/120902. 문자열 계산하기/문자열 계산하기.py
def solution(my_string):
    #문자열을 공백을 기준으로 분리합니다.
    
    tokens = my_string.split()
    
    result = int(tokens[0])
    
    
    for i in range(1, len(tokens), 2):
        operator = tokens[i]
        number = int(tokens[i + 1])
        #연산자에 따라 연산을 수행합니다.
        if operator == "+":
            result += number
        elif operator == '-':
            result -= number
    return result

File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.java
class Solution {
    public int solution(String my_string) {
        String token[] = my_string.split(" ");
        int answer = Integer.parseInt(token[0]);
        for(int i=1;i<token.length;i+=2){
            String operator = token[i];
            int nextNumber = Integer.parseInt(token[i+1]);
        
        if(operator.equals("+")){
            answer += nextNumber;
        }else if(operator.equals("-")){
            answer -= nextNumber;
        }
        }
        return answer;
    }
}